### DISCUSSION TITLE
whitelist

### DISCUSSION THREAD

**Discussion by yusufeweqewew** [IBAX-io/go-ibax#760] [Soru Cevap - Türkçe]
şimdi testnet'e katılanlara ileride whitelist verilebilir mi ?

**Comment by oze1** [comment]
> şimdi testnet'e katılanlara ileride whitelist verilebilir mi ?

testnete nasil katiliyoruz

**Reply by Yusufknk** [reply]
İlerleyen aşamalarda bu konu hakkında bilgilendirme yapılacaktır.
